src/main/java/address/events/LoadPrefsRequestEvent.java

```diff
@@ -0,0 +1,20 @@
+package address.events;
+
+import java.io.File;
+
+/**
+ * Indicates a request to load prefs from the file
+ */
+public class LoadPrefsRequestEvent extends BaseEvent {
+    /** The file from which the prefs to be loaded */
+    public File file;
+
+    public LoadPrefsRequestEvent(File file) {
+        this.file = file;
+    }
+
+    @Override
+    public String toString(){
+        return "from " + file.getAbsolutePath();
+    }
+}
```

Comment:
If loading preferences from the file happens only at the startup, we don't need an event for it. The MainApp can load the UserPrefs object using StorageManager (a static method?) and pass it to the constructor of ModelManager.